### PR TITLE
tomee-plume 8.0.8

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/tomee-plume.rb [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
-  mirror "https://archive.apache.org/dist/tomee/tomee-8.0.4/apache-tomee-8.0.4-plume.tar.gz"
-  sha256 "fff4d98606f023b11c37bb7b4ceb74600552ff2953dc90fb484456236da34663"
+  url "https://www.apache.org/dyn/closer.lua?path=tomee/tomee-8.0.8/apache-tomee-8.0.8-plume.tar.gz"
+  mirror "https://archive.apache.org/dist/tomee/tomee-8.0.8/apache-tomee-8.0.8-plume.tar.gz"
+  sha256 "396e92760483ac871a446219fa7f35b01927e4bea41bc9d922d0211bb526913b"
   license "Apache-2.0"
 
+  livecheck do
+    url :stable
+  end
```
Comment (carlocab) [inline Formula/tomee-plume.rb:11]: Isn't this the default?

**Review by carlocab — COMMENTED** [review]
Doesn't seem to be the latest release on the website yet. Not sure if that's just delayed.

**Comment by EricFromCanada** [comment]
It's listed here: https://tomee.apache.org/download.html

**Review by carlocab — APPROVED** [review]
Oops; misread the download page.

**Pull request closed without merging**